x-pack/plugins/secops/public/components/page/network/network_dns_table/translations.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const TOP_DNS_DOMAINS = i18n.translate('xpack.secops.networkDnsTable.title', {
+  defaultMessage: 'Top DNS Domains',
+});
+
+export const TOOLTIP = i18n.translate('xpack.secops.networkDnsTable.helperTooltip', {
+  defaultMessage: 'Only show DNS traffic; You can also sort any columns in the table below;',
```

Comment:
How about this plus your column sorting tip.

> "This shows DNS protocol traffic only, and can be useful for hunting domains used in DNS data exfiltration."

Or if that is too long then this shorter message:

> "This shows DNS protocol traffic only."

Reply:
I like the first one!!!